src/PureCloud.Client/Contracts/IQualityApi.cs

```diff
@@ -27,4 +27,96 @@ public interface IQualityApi
     Task<Calibration> DeleteQualityCalibrationAsync(string calibrationId, string calibratorId, CancellationToken cancellationToken = default);
 
     Task<EvaluationResponse> DeleteQualityConversationEvaluationAsync(string conversationId, string evaluationId, string expand = null, CancellationToken cancellationToken = default);
+
+    Task<AsyncQueryStatus> GetAnalyticsEvaluationsAggregatesJobAsync(string jobId, CancellationToken cancellationToken = default);
```

Comment:
Document all the methods in the interface along with their parameters. Also add new line at the end of each file.